apps/src/gamelab/GameLab.js

```diff
@@ -582,6 +582,30 @@ GameLab.prototype.afterInject_ = function(config) {
         'levelFailure'
       ].join(',')
     );
+    Blockly.JavaScript.addReservedWords(
```

Comment:
Based on the amount of hard-coded values, I wonder if we could move these to a constants.js file for blockly or gamelab to keep this file as readable as possible.